src/util/rocksdb/engine_metrics.rs

```diff
@@ -850,6 +852,17 @@ pub fn flush_engine_properties(engine: &DB, name: &str) {
                 .with_label_values(&[name, cf])
                 .set(pending_compaction_bytes as f64);
         }
+
+        // Compression ratio at levels
+        let opts = engine.get_options_cf(handle);
+        for level in 0..opts.get_num_levels() {
```

Comment:
Some levels maybe empty, is it ok?

Reply:
Yep, the compression ratio of an empty level will be -1.0.